stellar/docs/link-token.md

```diff
@@ -0,0 +1,230 @@
+
+- `LOCK_UNLOCK` (2): For tokens that are locked on the source chain and unlocked on the destination chain
+- `MINT_BURN` (4): For tokens that are burned on the source chain and minted on the destination chain
+
+## Parameters
+
+**Required:**
+
+- `salt`: Unique identifier for the token linking operation
+- `tokenAddress`: Address of the token to be linked
+- `destinationChain`: Name of the destination chain
+- `destinationTokenAddress`: Address of the token on the destination chain
+- `type`: Token manager type (LOCK_UNLOCK, MINT_BURN)
+
+**Optional:**
+
+- `--operator`: Operator address for the token manager on the destination chain
+- `--gas-token-address`: Gas token address (defaults to XLM on Stellar)
+- `--gas-amount`: Gas amount for cross-chain operations
+
+## Step-by-Step Process
+
+### Step 1: Setup Tokens
+
+Setup Token with decimals on both chains:
+
+**Chain A (Stellar):**
+
+```bash
+ts-node stellar/its.js deploy-interchain-token <name> <symbol> <decimal> <salt> <initialSupply>
+```
+
+**Chain B (EVM):**
+
+```bash
+ts-node evm/interchainTokenFactory.js \
+  --action deployInterchainToken \
+  --minter <minterAddress> \
+  --name <name> \
+  --symbol <symbol>
+  --decimals <decimal>
+  --initialSupply <initialSupply>
+  --salt <salt>
+  -n <network>
+```
+
+### Step 2: Register Token Metadata
+
+**Chain A (Stellar):**
+
+```bash
+ts-node stellar/its.js register-token-metadata <tokenAddress> --gas-amount <gasAmount>
+```
+
+**Chain B (EVM):**
+
+```bash
+ts-node evm/its.js --action registerTokenMetadata \
+  --tokenAddress <tokenAddress>
+  -n <network>
+```
+
+### Step 3: Register Custom Token
+
+Register the token on the source chain (Stellar):
+
+```bash
+ts-node stellar/its.js register-custom-token <salt> <tokenAddress> <tokenManagerType>
+```
+
+### Step 4: Link Token
+
+Link the token to the destination chain:
+
+```bash
+ts-node stellar/its.js link-token <salt> <destinationChain> <destinationTokenAddress> <tokenManagerType> \
+   --gas-amount <amount> \
+   --operator <operator>
+```
+
+## Examples
+
+### Example 1: Link USDC Tokens with LOCK_UNLOCK
+
+Link USDC tokens with different decimals (7 decimals on Stellar, 18 decimals on EVM):
+
+```bash
+# USDC already exists on both chains
+
+# Register USDC metadata on Stellar (7 decimals)
+ts-node stellar/its.js register-token-metadata CB64D3G...USDC --gas-amount 10000000
+
+# Register USDC metadata on EVM (18 decimals)
+ts-node evm/its.js --action registerTokenMetadata --tokenAddress 0xa0b86a33...USDC -n evm_chain
+
+# Register custom token on Stellar
+ts-node stellar/its.js register-custom-token 0x1234 CB64D3G...USDC LOCK_UNLOCK
+
+# Link token to EVM
+ts-node stellar/its.js link-token 0x1234 evm_chain 0xa0b86a33...USDC LOCK_UNLOCK --gas-amount 10000000
+
+# Interchain Token Transfer
+ts-node stellar/its.js interchain-transfer <tokenId> evm_chain <destinationAddress> <amount> --gas-amount 10000000
+```
+
+### Example 2: Link Custom Token with MINT_BURN
```

Comment:
not a very useful example, since most cmd args are stubbed out. one example is sufficient